ui/ducks/app/app.ts

```diff
@@ -239,6 +241,18 @@ export default function reduceApp(
         deprecatedNetworkModalOpen: false,
       };
 
+    case actionConstants.CONNECT_ACCOUNTS_MODAL_OPEN:
```

Comment:
Will this "Connect Accounts" modal be used anywhere else?  Can it just be shown/hidden locally via `setState`?

Reply:
Upated